### PR TITLE
Cleanup network testing according new HTTP API - Closes #283

### LINKED ISSUE #283: Cleanup network testing according new HTTP API
### Description

The current network testing code is outdated according v5 SDK changes. The endpoints have changed, we need to cleanup the network testing corresponding to previous version to be able to continue testing network.

### Acceptance Criteria

- Remove all network related tests
- Introduce new network tests according to latest HTTP Api changes

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #283 

### How was it solved?

- Cleaned up QA repo
- Added network test under test folder

### How was it tested?

- Run a node (forging)
- Install and Run `NETWORK=devnet ts-node test/scripts/stress_test.ts`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,6 +72,7 @@
 		"axios": "0.19.2"
 	},
 	"devDependencies": {
+		"@liskhq/lisk-codec": "0.1.0-debug.1",
```
Comment (shuse2) [inline package.json:75]: codec is exposed from `lisk-sdk` already

File: test/network_test/index.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+/* eslint-disable no-console */
+
```
Comment (shuse2) [inline test/network_test/index.ts:15]: prefer not to use `index.ts`

File: test/network_test/index.ts [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+		);
+	}
+
+	// const accountToReclaim: CreateAccount = createAccount();
+	// await sendReclaimTransactions(channel, nodeInfo, accountToReclaim);
+	// console.log('\n');
+	// await wait();
+
+	console.info('Finished!!');
+	process.exit(0);
```
Comment (shuse2) [inline test/network_test/index.ts:156]: probably no need to `process.exit`?

File: test/network_test/accounts.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
+/*
```
Comment (shuse2) [inline test/network_test/accounts.ts:1]: since this will be used against local node, `test/scripts` folder is fine?

**Review by shuse2 — COMMENTED** [review]
LGTM in general

File: test/network_test/index.ts (hunk 2 above)
Comment (ManuGowda) [reply test/network_test/index.ts:15]: if not index then what to use? network_test.ts?

File: package.json (hunk 1 above)
Comment (ManuGowda) [reply package.json:75]: i think codec Schema is not exposed?

Comment (shuse2) [reply package.json:75]: We should expose it through SDK. I will add it to https://github.com/LiskHQ/lisk-sdk/issues/5565

File: test/scripts/utils/transactions/send.ts [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+
+	await handleTransaction(channel, transaction, 'transfer transaction from multisig account');
+};
+
+export const sendReclaimTransactions = async (
+	channel: IPCChannel,
+	nodeInfo: Record<string, unknown>,
+	fromAccount: PassphraseAndKeys,
+) => {
+	const { networkID } = nodeInfo as { networkID: string };
```
Comment (shuse2) [inline test/scripts/utils/transactions/send.ts:233]: nodeInfo has only `networkIdentifier`?

Comment (ManuGowda) [reply test/scripts/utils/transactions/send.ts:233]: no .. i only needed networkID 😄

**Review by shuse2 — APPROVED** [review]

File: test/scripts/stress_test.ts [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+import { IPCChannel, systemDirs } from 'lisk-sdk';
+import { PassphraseAndKeys, createAccount, genesisAccount } from './utils/accounts';
+import {
+	sendTokenTransferTransactions,
+	sendDelegateRegistrationTransaction,
+	sendVoteTransaction,
+	getBeddows,
+	sendMultiSigRegistrationTransaction,
+	sendTransferTransactionFromMultiSigAccount,
+	// sendReclaimTransactions,
```
Comment (mitsuaki-u) [inline test/scripts/stress_test.ts:25]: should this be commented?

Comment (ManuGowda) [reply test/scripts/stress_test.ts:25]: once this issue is addressed, we can uncomment them https://github.com/LiskHQ/lisk-core/issues/345. I added the comment below as well.

**Review by mitsuaki-u — APPROVED** [review]

**Pull request merged**